nexus/types/src/external_api/params.rs

```diff
@@ -29,6 +29,9 @@ pub struct SiloCreate {
     pub discoverable: bool,
 
     pub user_provision_type: shared::UserProvisionType,
+
+    /// if set, this group will be created and granted the silo admin role
```

Comment:
```suggestion
    /// If set, this should match the name of a group stored in the Identity Provider.  This group will be granted the "Silo Admin" role on the new Silo.  The SAML group attribute must also be configured.
```

(this is public-facing API docs)

Reply:
expanded in 7837edc0